tox.ini

```diff
@@ -0,0 +1,12 @@
+[tox]
+envlist = py27, py36, py38
```

Comment:
Just one thing. We check 3 versions but in setup.py we list all from 2 to 3.6. So maybe we should adjust it.